cost/azure/instances_log_analitics_utilization/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## v2.1
 
-- Fixed bug where Azure returns instance details without MemoryGB field
```

Comment:
I think this version might be 2.2.  2.1 already exists

Reply:
Done!